test/Tester/RelationalUtilities/RelationalStorageForTesting.cs

```diff
@@ -17,11 +17,11 @@ public abstract class RelationalStorageForTesting
                 {AdoNetInvariants.InvariantNameMySql, cs => new MySqlStorageForTesting(cs)}
             };
 
-        private readonly IRelationalStorage storage;
```

Comment:
this should remain `private readonly`. relevant scripts should be added to 'MySqlStorageForTesting' and 'SqlServerStorageForTesting'